[Workers] Cache revision (#2688)
cloudflare/cloudflare-docs: products/workers/src/content/runtime-apis/cache.md

#### Invalid parameters

> `cache.put` throws an error if:

Reviewer: ```suggestion
`cache.put` will throw an error if:
```

>   - the `request` passed is a method other than `GET`.
>   - the `response` passed is a `status` of [`206 Partial Content`](https://httpstatuses.com/206).
>   - the `response` passed contains the header `Vary: *` (required by the Cache API specification).

Reviewer: ```suggestion
- The `request` passed is a method other than `GET`.
- The `response` passed has a `status` of [`206 Partial Content`](https://httpstatuses.com/206).
- The `response` passed contains the header `Vary: *` (required by the Cache API specification).
```

    - The string or [`Request`](/runtime-apis/request) object used as the lookup key. Strings are interpreted as the URL for a new `Request` object.

> - `options`

Author: @lukeed also this doesn't have a <Type> like `request` does, what would it be?

>     -  Can contain one possible property: `ignoreMethod` (Boolean). Consider the request method a GET regardless of its actual value.

Author: @lukeed should this be "considers"?

>     -  Can contain one possible property: `ignoreMethod` (Boolean). Consider the request method a GET regardless of its actual value.

Reviewer 2: ```suggestion
    -  Can contain one possible property: `ignoreMethod` (Boolean). When `true`, the request is considered to be a `GET` request regardless of its actual value.
```

--------------------------------

> ## More information

Reviewer: Our content strategy mentions a "Related resources" section, and I've been using this as the title for such a section.

> ## More information

Author: ```suggestion
## Related resources
```
